cmd/cert-checker/main.go

```diff
@@ -314,19 +316,23 @@ type config struct {
 		BadResultsOnly      bool
 		CheckPeriod         cmd.ConfigDuration
 
+		// TODO(): This branch is deprecated and can be removed once staging
```

Comment:
It looks like you intended an issue number with the TODO() to track the deprecation and future removal. If there isn't one, please file one.

Reply:
Ah of course I missed putting this in a spot, thanks!